### PR TITLE
Port: humble to ros2

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

File: ros_gz/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,12 @@ Changelog for package ros_gz
 * Merge remote-tracking branch 'origin/humble' into ahcorde/humble_to_ros2
 * Contributors: Alejandro Hernández Cordero, ahcorde
 
+0.244.10 (2023-05-03)
```
Comment (caguero) [inline ros_gz/CHANGELOG.rst:12]: Do these empty blocks make sense? Same in other changelogs

**Review by caguero — DISMISSED** [review]
It looks good to me besides a minor comment about the changelogs.

**Comment by caguero** [comment]
Is CI supposed to fail like this?

File: ros_gz/CHANGELOG.rst (hunk 1 above)
Comment (mjcarroll) [reply ros_gz/CHANGELOG.rst:12]: Yes, this is generally how it is done with ROS releases.  All packages in a repository have to be released at the same time, even if not all the packages have changes.  This is indicated with a blank changelog entry.

**Review by caguero — APPROVED** [review]

**Pull request merged**